ksqldb-functional-tests/src/test/resources/query-validation-tests/timestampsub.json

```diff
@@ -41,6 +41,23 @@
         {"topic": "TEST2", "key": 0, "value": {"VALUE": 5}},
         {"topic": "TEST2", "key": 0, "value": {"VALUE": -95}}
       ]
+    },
+    {
+      "name": "handles null values",
+      "statements": [
+        "CREATE STREAM TEST (ID INT KEY, time TIMESTAMP, num INTEGER) WITH (kafka_topic='test', value_format='AVRO');",
+        "CREATE STREAM TEST2 AS SELECT id, timestampsub(MILLISECONDS, num, time) AS VALUE FROM TEST;"
```

Comment:
Not too familiar with the new timestamp code, is milliseconds a reserved keyword in the syntax now for use cases like these?

Reply:
yup, its the same set of keywords that window expressions use